labrad/types.py

```diff
@@ -447,16 +453,13 @@ def reprLRData(s):
 
 # LabRAD type classes
 
-class LRType(object):
+class LRType(with_metaclass(RegisterParser, object)):
```

Comment:
This version is slightly different from the one in `six`; according to the docstring:
> This has the advantage over six.with_metaclass of not introducing dummy classes into the final MRO.